mobile-app/app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx

```diff
@@ -20,7 +20,7 @@ export function NetworkDetails (): JSX.Element {
     masternodeCount,
     lastSuccessfulSync
   } = useSelector((state: RootState) => state.block)
-  const syncFormattedDate = (lastSuccessfulSync != null) ? dayjs(lastSuccessfulSync).format('MMM D, h:mm a') : ''
+  const syncFormattedDate = (lastSuccessfulSync != null) ? dayjs(lastSuccessfulSync).format('lll') : ''
```

Comment:
These are not the same format

Reply:
Indeed, but there is no equivalent localized format and I think it is close enough.
See https://day.js.org/docs/en/display/format